lib/proxy_api/version.rb

```diff
@@ -11,6 +11,10 @@ def proxy_versions
       raise ProxyException.new(url, e, N_("Unable to detect version"))
     end
 
+    def plugin_versions
+      proxy_versions["modules"]
```

Comment:
since you are calling proxy_versions multiple times, cache it, e.g.

```ruby
    def proxy_versions
      @version_response ||= parse(get)
    rescue => e
      raise ProxyException.new(url, e, N_("Unable to detect version"))
    end